### ISSUE TITLE
Hello, World

### ISSUE THREAD

**Issue by gabriel-aplok** [gabriel-aplok/gabriel-aplok.github.io#1]
Hello!

**Comment by gabriel-aplok** [comment]
Test

**Comment by gabriel-aplok** [comment]
This project is still in **development**, so doesn't have all the features I intend to add, please, I ask for your understanding and give good feedback! 💙

**Comment by gabriel-aplok** [comment]
Don't forget to also download **Mini Apocalypse** to create the best addon pack!
https://gabriel-aplok.github.io/projects/mini-apocalypse.html

**Comment by gabriel-aplok** [comment]
Hello, world! 🤓☝️

**Comment by BananaOTime** [comment]
Can I use this addon for a smp I’m making?

**Comment by gabriel-aplok** [comment]
> Can I use this addon for a smp I’m making?
> 
> (also just want to say this is one of the best gun addons out there, and I hope you keep updating it👍)

Yes, absolutely! You just need to give me clear credits, thank you! ❤️

**Comment by BananaOTime** [comment]
> > Can I use this addon for a smp I’m making?
> > (also just want to say this is one of the best gun addons out there, and I hope you keep updating it👍)
> 
> Yes, absolutely! You just need to give me clear credits, thank you! ❤️

Thank you, and I will give full credit

**Comment by Argongid** [comment]
the mod are execelent but in 1.20.41 we can't recharge it. Sorry for the text I don't speak english.

**Comment by gabriel-aplok** [comment]
> the mod are execelent but in 1.20.41 we can't recharge it. Sorry for the text I don't speak english.

Why u can't reload? I'm able to reload normally here and the majority of people who play the mod :(

**Comment by MakmakGamer09** [comment]
Hello, can I use your Addon to make my own map and to publish?

**Comment by gabriel-aplok** [comment]
> Hello, can I use your Addon to make my own map and to publish?

Sure, I would appreciate if u "adverse" the project, but sure!

**Comment by BananaOTime** [comment]
I just wanted to ask if I could edit some things in the addon, I don’t plan on changing much?

**Comment by gabriel-aplok** [comment]
> do you have any other website links for the 1.07 update like work.ink? I am unable to use linkvertise.

MCPEDL and MODBAY have others working links without linkvertise, searh on this platforms.

**Comment by Epic-bot420** [comment]
Awesome mod. But i do think it could use a RPG and the Fn Fal shouldbbe semi automatic. Keep up the awesome work though:D

**Comment by pupsik2211** [comment]
Addon is amazing, it one of the best gun mods today. And i have one question, in the future its planed to add tactiс reloading?

**Comment by fabriciocon** [comment]
Hello, can I use your Addon to make my own map and to publish?

**Comment by LEichert07** [comment]
> Comments for Aplok Guns

Hey Aprok. Your mod has greatly increased the quality of my nations server (credits at spawn if your wondering) but we can't seem to make the attachments work. I read the blog and we tried using the attachments as described but they don't seem to be working for us, any help is greatly appreciated
